### DISCUSSION TITLE
New Apple Silicon build folder issues

### DISCUSSION THREAD

**Discussion by sfmike64** [transmission/transmission#2884] [General]
I downloaded the latest Universal build for Mac (7813) and it constantly asks me to give permission to access my Downloads folder and the network folder where my downloaded files are stored. 

I recall this being normal behavior once. But this build (unlike the intel native general release 3.0) keeps asking over and over. It's particularly annoying as the dialog box comes to the forefront so whatever else you're doing is interrupted.

**Comment by GaryElshaw** [comment]
If you go to the security and privacy panel in system prefs and give Transmission folder or disk access it should stop that. Worth stopping by the firewall pref and giving it access there too!

**Reply by tollfree** [reply]
Didn't help.
The app asked for access to Downloads folder and Incoming connections on start. I gave it, the app continues to ask for these permissions many times any time it started.
Tried to remove it from Firewall options and Files and Folders lists and add it again — no effect.

**Comment by sfmike64** [comment]
Thanks. It's already set there.

**Comment by sjackson1236** [comment]
hi I have this issue as well, right now it asks me to set permissions four times every time I start up the program, even though I already did it. using an m2 air, ventura 13.0.1
